src/dm/dm_policy.cpp

```diff
@@ -182,7 +182,7 @@ void dm_policy_t::operator = (const dm_policy_t& obj)
     this->m_policy.sta_traffic_stats = obj.m_policy.sta_traffic_stats;
     this->m_policy.sta_link_metric = obj.m_policy.sta_link_metric;
     this->m_policy.sta_status = obj.m_policy.sta_status;
-    strncpy(this->m_policy.managed_sta_marker, obj.m_policy.managed_sta_marker, strlen(obj.m_policy.managed_sta_marker) + 1);
+    memmove(this->m_policy.managed_sta_marker, obj.m_policy.managed_sta_marker, strlen(obj.m_policy.managed_sta_marker) + 1);
```

Comment:
Similar to previous `memmove` suggestion:
```suggestion
    memcpy(this->m_policy.managed_sta_marker, obj.m_policy.managed_sta_marker, sizeof(em_long_string_t));
```